### PR TITLE
Add AWorldOfPainFO3

### PR DESCRIPTION
**Pull request by Superfly-Johnson**
Also found an incompatibility with fafnir's unique in xEdit, fixed it and posted the patch, which is also in this commit. Minor stuff, just formid lists needing to be merged. :)

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -2472,6 +2472,26 @@ plugins:
         subs: [ 'Low Robots ESP' ]
         condition: 'many("AWOP(Very)?LowRobots\.esp")'
 
+  - name: 'AWorldOfPainFO3.esm'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/66265'
```
Comment (pStyl3) [inline masterlist.yaml]: Please add a trailing slash `/` at the end of the link.

File: masterlist.yaml [hunk 2]
```diff
@@ -2472,6 +2472,26 @@ plugins:
...
+        name: 'TTW A World of Pain for Fallout 3'
+    req: [ *TTWv320 ]
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Literally Who''s Patch Emporium'
+          - '[ AWorldOfPainFO3 - FafnirUniqueTTWOverhaul Patch ](https://www.nexusmods.com/newvegas/mods/83169/)'
```
Comment (pStyl3) [inline masterlist.yaml]: The empty spaces before and after `AWorldOfPainFO3 - FafnirUniqueTTWOverhaul Patch` (so within the brackets `[ ]`) can be removed.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Review by pStyl3 — APPROVED** [review]

File: masterlist.yaml [hunk 3]
```diff
@@ -2472,6 +2472,28 @@ plugins:
...
+          - '[AWorldOfPainFO3 - FafnirUniqueTTWOverhaul Patch](https://www.nexusmods.com/newvegas/mods/83169/)'
+        condition: 'active("FafnirUniqueTTWOverhaul.esp") and not active("AWorldOfPainFO3-FafnirUniqueTTWOverhaul.esp")'
+      - <<: *patchProvided
+        subs: [ 'JIP Companions Command and Control' ]
+        condition: 'active("JIP Companions Command & Control.esp") and not file("textures/jazzisparis/ccc/avatar_the_warden.dds")'
+    tag:
+      - C.Light
+      - Faction
+    clean:
+    # v2.07
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**